docs/coding-projects/hello-world.md

```diff
@@ -0,0 +1,384 @@
+    'tutorials',
+    'hello world tutorial',
+    'pact hello world',
+    'smart contract deployment',
+  ]
+---
+
+# Hello, World! revisited
+
+In the [Quick start](../quickstart), you were introduced to a simple `greeting` module. 
+Modules provide the basic foundation for all Pact smart contracts. 
+In fact, the term `module` is a reserved keyword that you use to define and install a module with a specified _name_, 
+
+The `greeting` module defined a single function—one very much like the traditional Hello, World! program you see when learning any new programming language.
+
+In the `00-hello-world` project folder, you'll take a closer look at the `hello-world` module, extend its functionality, and deploy it on the local development network.
+
+## Before you begin
+
+Before starting this project, verify your environment meets the following basic requirements:
+
+- You have a GitHub account and can run `git` commands.
+- You have installed the Pact programming language and command-line interpreter.
+- You have installed the `kadena-cli` package and have a working directory with initial configuration settings.
+- You have a local development node that you can connect to that runs the `chainweb-node` program, either in a Docker container or on a physical or virtual computer.
+
+If you have these basics covered, you're ready to go.
+
+## Get the starter code
+
+To get started:
+
+1. Open a terminal shell on your computer.
+
+2. Clone the `pact-coding-projects` repository by running the following command:
+   
+   ```bash
+   git@github.com:kadena-docs/pact-coding-projects.git
+   ```
+
+3. Change to the `00-hello-world` directory by running the following command:
+   
+   ```bash
+   cd pact-coding-projects/00-hello-world
+   ```
+
+4. Open the `hello-world.pact` file in your code editor.
+   
+   ```pact
+   ;;
+   ;; "Hello, world!" module
+   ;; 
+   ;;-----------------------------------------------------------------------
+   ;;
+   ;;  Use semicolons (;) for comments in smart contracts.
+   ;;  By convention, use:
+   ;;  
+   ;;  - A single semicolon (;) for short notes on a single line of code. 
+   ;;  - Two semicolons (;;) to describe functions or other top-level forms.
+   ;;  - Three semicolons (;;;) to separate larger sections of code.
+   ;;
+   ;;-----------------------------------------------------------------------
+   
+   (module helloWorld GOVERNANCE
+       "You can also embed comments in smart contracts by using quoted strings."
+   
+       (defcap GOVERNANCE () true)
+         (defun say-hello(name:string)
+           (format "Hello, {}! ~ from Kadena" [name])
+       )
+   )
+   ```
+
+   A few things you should note about this starter code:
+   
+   - The `module` being defined is named `helloWorld`.
+   - Every module is governed by either a _keyset_ or a _capability_. 
+     In this example, the GOVERNANCE capability is used to control access to the module.
+     This capability must evaluate to true to grant access.
```

Comment:
This is incorrect. The module guard has to successfully be executed, i.e. needs to be restricted by `enforce-*` @emilypi maybe you can comment here?